### PR TITLE
Improved visibility on collapse button in Java Lab Preview Pane

### PR DESCRIPTION
**Pull request by AfifahK**
## Summary
Updated the color of Java Lab Preview Pane Collapse Button to white in order for it to be more visible. 
Before:
image
After:
image

## Links
- jira ticket: [https://codedotorg.atlassian.net/browse/SL-1006]()

## Testing story
Manual tests to see whether the color visibly changed. Ensured that the collapse button on the instruction panel above remained the original color. 

See photo: 
image

## Privacy
This change does not involve the collection, use, or sharing of both new and existing Personal Data. 

## PR Checklist
- [] Tests provide adequate coverage
- [] Privacy and Security impacts have been assessed
- [] Code is well-commented
- [] New features are translatable or updates will not break translations
- [] Relevant documentation has been added or updated
- [] User impact is well-understood and desirable
- [] Pull Request is labeled appropriately
- [] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/PreviewPaneHeader.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import PaneHeader, {
 } from '@cdo/apps/templates/PaneHeader';
 import CollapserIcon from '@cdo/apps/templates/CollapserIcon';
 import {RecordingFileType} from '../code-studio/components/recorders';
+import {white} from '../util/color';
```
Comment (bencodeorg) [inline apps/src/javalab/PreviewPaneHeader.jsx]: Nit: I had no idea you could import individual colors this way! I do see one other example of us importing colors like this (ie, grabbing a specific color), but the more common approach would just be to import the color object like this, then referencing the specific color in your style object:

https://github.com/code-dot-org/code-dot-org/blob/78190b48869febecc5d575787ab0eb05bbce78f0/apps/src/javalab/JavalabConsole.jsx#L5

File: apps/src/javalab/PreviewPaneHeader.jsx [hunk 2]
```diff
@@ -22,7 +23,12 @@ export default function PreviewPaneHeader({
       <PaneSection className={'pane-header-section pane-header-section-left'}>
         <PaneButton
           headerHasFocus
-          icon={<CollapserIcon isCollapsed={isCollapsed} />}
+          icon={
+            <CollapserIcon
+              isCollapsed={isCollapsed}
+              style={styles.previewPaneCollapserIcon}
```
Comment (bencodeorg) [inline apps/src/javalab/PreviewPaneHeader.jsx]: Nit: this style is exclusively used within our preview pane header, so maybe the `previewPane` prefix isn't necessary? Maybe just `collapserIcon`?

**Review by bencodeorg — APPROVED** [review]
🎉 A couple of nits but overall looks good to me!

**Review by fisher-alice — APPROVED** [review]
Nice job Afifah! I appreciate that you added before/after screenshots and +1 to Ben's comments.

One thing to look out for in UI changes is differences in eyes tests on `test` - so when you merge a PR that entails UI changes, you may want to keep a look out on  #infra-test. Usually the DOTD calls out if there are any differences and baselines can be changed accordingly. Probably won't happen for this PR, but just a FYI that these tests are not run on drone builds.

Yay for your first Code.org PR! 🎉 !

File: apps/src/code-studio/components/AdvancedShareOptions.jsx [hunk 3]
```diff
@@ -38,6 +38,12 @@ const style = {
     color: color.neutral_dark,
     fontFamily: "'Gotham 5r', sans-serif",
   },
+  warningp: {
```
Comment (fisher-alice) [inline apps/src/code-studio/components/AdvancedShareOptions.jsx]: Could you explain why you added this to the PR? Thanks!

Comment (fisher-alice) [reply apps/src/code-studio/components/AdvancedShareOptions.jsx]: Just looked at your other [open PR](https://github.com/code-dot-org/code-dot-org/pull/53106) and I think this change may belong to that one? If so, you can just revert [this commit](https://github.com/code-dot-org/code-dot-org/pull/53057/commits/249d74f7b4bc2e15d8f925e2d5cd13a2cc704758).

**Pull request merged**